tornettools/tornettools

```diff
@@ -223,6 +223,12 @@ def main():
         formatter_class=my_formatter_class)
     generate_parser.set_defaults(func=generate, formatter_class=my_formatter_class)
 
+    generate_parser.add_argument('--hidden',
+        help="""Servers will be generated as hidden services rather than clearnet servers,
+            and client traffic will be directed accordingly""",
+        action="store_true", dest="hidden",
+        default=False)
```

Comment:
I think ultimately we would want to change this to use a fraction between 0.0 and 1.0 to represent the fraction of our tgen servers that are available as onion services, and then maybe a separate param for the fraction of clients that fetch onion service traffic. I think this would be more generally useful than an all-or-nothing flag.

Reply:
Make sense. Done.